lib/processor.js

```diff
@@ -58,14 +61,23 @@ function preprocess(text) {
 }
 
 /**
+ * Excludes unsatisfiable rules from the list of messages.
+ * @param {Message} message A message from the linter.
+ * @returns {boolean} True if the message should be included in output.
+ */
+function excludeUnsatisfiableRules(message) {
+    return UNSATISFIABLE_RULES.indexOf(message.ruleId) < 0;
```

Comment:
Do we wanna follow the core conventions for the plugin? My feeling is plugins should not be bound by what core supports or not.